src/styles/price.css

```diff
@@ -0,0 +1,24 @@
+.card__price {
```

Comment:
create separate files only bem block
the same here
move styles to appropriate files